pkg/build/rpm_test.go

```diff
@@ -83,48 +227,54 @@ func TestCopyRPMs(t *testing.T) {
 	_, err = os.Stat(filepath.Join(builder.context.CombustionDir, "rpm2.rpm"))
 	require.NoError(t, err)
 
-	// Cleanup
-	assert.NoError(t, file1.Close())
-	assert.NoError(t, file2.Close())
+	expectedFilename := filepath.Join(builder.context.CombustionDir, modifyRPMScriptName)
+	foundBytes, err := os.ReadFile(expectedFilename)
+	require.NoError(t, err)
+
+	foundContents := string(foundBytes)
+	assert.Contains(t, foundContents, "rpm1.rpm")
+	assert.Contains(t, foundContents, "rpm2.rpm")
 }
 
-func TestGetRPMFileNamesNoRPMs(t *testing.T) {
+func TestGenerateRPMPath(t *testing.T) {
 	// Setup
-	context, err := NewContext("../config/testdata", "", true)
+	tmpDir, err := os.MkdirTemp("", "eib-generate-RPM-path-test-")
+	require.NoError(t, err)
+	defer os.RemoveAll(tmpDir)
+
+	expectedPath := filepath.Join(tmpDir, "rpms")
+	err = os.Mkdir(expectedPath, 0o755)
+	require.NoError(t, err)
+
+	context, err := NewContext(tmpDir, "", true)
 	require.NoError(t, err)
 	defer func() {
 		assert.NoError(t, CleanUpBuildDir(context))
 	}()
 
-	builder := &Builder{
-		context: context,
-	}
-
-	rpmSourceDir := filepath.Join(context.ImageConfigDir, "rpms")
+	builder := Builder{context: context}
 
 	// Test
-	rpmFileNames, err := builder.getRPMFileNames(rpmSourceDir)
+	generatedPath, err := builder.generateRPMPath()
 
 	// Verify
-	require.ErrorContains(t, err, "no rpms found")
+	require.NoError(t, err)
 
-	assert.Empty(t, rpmFileNames)
+	assert.Equal(t, expectedPath, generatedPath)
 }
 
-func TestCopyRPMsNoRPMDir(t *testing.T) {
+func TestGenerateRPMPathNoRPMDir(t *testing.T) {
```

Comment:
Does the test address the expectation that the name applies?

Reply:
I thought so, but I updated it just to be sure by adding a temporary directory that doesn't contain `/rpms`